vagrant_box_defaults.rb

```diff
@@ -5,7 +5,7 @@
 $SERVER_BOX = "cilium/ubuntu-dev"
 $SERVER_VERSION= "173"
 $NETNEXT_SERVER_BOX= "cilium/ubuntu-next"
-$NETNEXT_SERVER_VERSION= "56"
+$NETNEXT_SERVER_VERSION= "59"
```

Comment:
+3 already! :scream:

Reply:
There was an issue in the packer builder on jenkins that @nebril recently fixed.